polarization/tissueLayer.py

```diff
@@ -65,18 +62,52 @@ def propagateManyThrough(self, vectors: List[JonesVector]) -> List[JonesVector]:
         return J
 
     def backscatter(self, vector: JonesVector) -> JonesVector:
-        return self.backscatterTransferMatrix(vector.k) * vector
+        deltaX, deltaY = self.scatteringDeltaAt(vector.k)
+        M = JonesMatrix(A=deltaX, B=0, C=0, D=deltaY, orientation=self.orientation)
+        return M * vector
 
-    def backscatterMany(self, vectors: List[JonesVector]):
+    def backscatterMany(self, vectors):
```

Comment:
Weird API, could we have something where vectors is always the same type? or equivalent type?